include/jemalloc/internal/jemalloc_internal_defs.h.in

```diff
@@ -440,4 +440,7 @@
 /* If defined, realloc(ptr, 0) defaults to "free" instead of "alloc". */
 #undef JEMALLOC_ZERO_REALLOC_DEFAULT_FREE
 
+/* If defined, use rdtscp to get the time stamp and the processor ID. */
```

Comment:
nit: I wouldn't say `use rdtscp to ...` (since it's not the most preferred approach with the current logic). Maybe `support the use of rdtscp to ...`?

Reply:
True, `support the use of rdtscp ...` looks more accurate. Updated.